media-api/app/controllers/MediaApi.scala

```diff
@@ -306,12 +307,29 @@ object MediaApi extends Controller with ArgoHelpers {
     Link("related-labels", uri)
   }
 
+  private val getSuggestedLabelsLink =
+    Link("suggested-labels", s"$rootUri/suggest/edits/labels")
+
   def suggestMetadataCredit(q: Option[String], size: Option[Int]) = Authenticated.async { request =>
     ElasticSearch
       .completionSuggestion("suggestMetadataCredit", q.getOrElse(""), size.getOrElse(10))
       .map(c => respondCollection(c.results))
   }
 
+  def suggestLabels() = Authenticated {
+    val pseudoFamousLabels = List(
```

Comment:
Sad to hardcode in the codebase :crying_cat_face: 

Could it be an application config thing?

I'd start with it hardcoded to prove the point though, so :+1: